src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -107,86 +111,89 @@ const CreateConceptForm = props => (
-          <option>Time</option>
-          <option>Datetime</option>
-          <option>Boolean</option>
-          <option>Rule</option>
-          <option>Structured-Numeric</option>
-          <option>Complex</option>
-          <option>Coded</option>
-        </select>
-      </div>
+          )}
+        </div>
+        <div className="datatypefield">
+          <label htmlFor="datatype">Datatype</label>
+          <select
+            id="datatype"
+            name="datatype"
+            required
+            value={props.state.datatype}
+            className="form-control "
+            onChange={props.handleChange}
+          >
+            <option>Numeric</option>
+            <option>Text</option>
+            <option>None</option>
+            <option>Document</option>
+            <option>Date</option>
+            <option>Time</option>
+            <option>Datetime</option>
+            <option>Boolean</option>
+            <option>Rule</option>
+            <option>Structured-Numeric</option>
+            <option>Complex</option>
+            <option>Coded</option>
+          </select>
+        </div>
 
-      {props.concept.toString().trim() === 'question' ?
-        <div className="form-group answer">
-          <div className="row col-12 custom-concept-list">
-            <h6 className="text-left section-header">Answers</h6>
-            <AnswersTable {...props} />
-            <a href="#!" className="text-left add-more-answers" id="add-more-answers" onClick={props.addAnswer}>
-            Add a new answer mapping
-            </a>
+        {concept.toString().trim() === 'question' ? (
+          <div className="form-group answer">
+            <div className="row col-12 custom-concept-list">
+              <h6 className="text-left section-header">Answers</h6>
+              <AnswersTable
+                handleAsyncSelectChange={handleAsyncSelectChange}
+                queryAnswers={queryAnswers}
+                selectedAnswers={selectedAnswers}
+                handleAnswerChange={handleAnswerChange}
+              />
+            </div>
           </div>
-        </div>
-      : null}
+        ) : null }
 
-      <div className="form-group">
-        <div className="row col-12 custom-concept-list">
-          <h6 className="text-left section-header">Names</h6>
-          <CreateConceptTable {...props} />
-          <a
-            href="#!"
-            className="text-left add-more-names"
-            id="add-more-name"
-            onClick={props.handleNewName}
-          >
+        <div className="form-group">
+          <div className="row col-12 custom-concept-list">
+            <h6 className="text-left section-header">Names</h6>
+            <CreateConceptTable {...props} />
+            <a
+              href="#!"
+              className="text-left add-more-names"
+              id="add-more-name"
+              onClick={props.handleNewName}
+            >
             Add another name...
```

Comment:
Also these

Reply:
@Tittoh once again, it's the linter issue there.